tcms/testcases/migrations/0006_merge_text_field_into_testcase_model.py

```diff
@@ -27,6 +32,13 @@ def forward_copy_data(apps, schema_editor):
     for test_case in test_case_model.objects.all():
         latest_text = test_case_text_model.objects.filter(case=test_case.pk).order_by('-pk').first()
         if latest_text:
+            file_name = '/tmp/kiwitcms-testcases-migrations-0006-\
+TestCaseText-%d' % latest_text.pk  # nosec:B108:hardcoded_tmp_directory
+            with open(file_name, 'w') as outfile:
+                print('>>> FORWARDS: dumping TestCaseText', latest_text.pk)
+                json.dump(Serializer(model=latest_text).serialize_model(), outfile)
+                print('=== DONE DUMPING')
```

Comment:
The print statements were added in my original implementation as a debugging and helper tool to see what's going on. We don't need them.